samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/BotController.cs

```diff
@@ -135,6 +140,7 @@ public class BotController : ControllerBase
     /// <returns>The serialized Bot object of the chat id.</returns>
     [Authorize]
     [HttpGet]
+    [ActionName("DownloadAsync")]
```

Comment:
Do we need to set ActionName while there is Route?

Reply:
Yeah originally I was going to pass DownloadAsync into CreatedAtAction in bot/upload so this attribute was required for correct action reference, but since I opted to pass the ChatHistoryController.GetChatSessionByIdAsync action instead, we technically don't need this. 

But I left it there because it was a pain to debug and will be helpful if someone wants to reference this method as an action in the future. Without the attribute explicitly defined like this, the method wasn't being discovered correctly, and led to this error: Error: 500: => System.InvalidOperationException: No route matches the supplied values.